spread/project.go

```diff
@@ -119,6 +119,9 @@ type System struct {
 	Password string
 	Workers  int
 
+	// Only for Linode and Google so far.
+	Storage  Size
```

Comment:
This setting should cascade from the backend when the backend has it set, so that the actual backend implementations don't need to check both. There are good examples in how other options are handled (Priority, Manual, etc).

Reply:
@niemeyer thanks for reviewing this, I already implemented the change.